Sources/SkipWeb/WebEngine.swift

```diff
@@ -384,6 +384,7 @@ public class WebEngineConfiguration : ObservableObject {
     @Published public var isScrollEnabled: Bool
     @Published public var pageZoom: CGFloat
     @Published public var isOpaque: Bool
+	@Published public var customUserAgent: String
```

Comment:
Is there any reason to not make this a nullable `String?`? That way you can check against `nil` rather than against a blank string…

Reply:
Sounds good.

Reply:
Wait, committed too soon. Hold please